python/pyhail/dataset.py

```diff
@@ -280,6 +379,71 @@ def annotate_samples_expr(self, condition):
+        phenotype without the `-q` flag will return an error (unless all values
+        happen to be ``0``, ``1``, ``2``, and ``-9``):
+
+        >>> vds.annotate_samples_fam("data/myStudy.fam", quantPheno=True)
+
+        Import case-control phenotype data from an
+        arbitrary-whitespace-delimited PLINK .fam file into sample annotations:
+
+        >>> vds.annotate_samples_fam("data/myStudy.fam", delimiter="\\s+")
+
+        **Annotation Schema**
+
+        The annotation names, types, and missing values are shown below,
+        assuming the default root ``sa.fam``.
+
+        +------------+-------------------+--------+------------+
+        |**Field**   |**Annotation**     |**Type**|**Missing** |
+        |            |                   |        |            |
+        +------------+-------------------+--------+------------+
+        |Family ID   |``sa.fam.famID``   |String  |``0``       |
+        |            |                   |        |            |
+        |            |                   |        |            |
+        +------------+-------------------+--------+------------+
+        |Sample ID   |``s``              |String  |            |
+        |            |                   |        |            |
+        |            |                   |        |            |
+        +------------+-------------------+--------+------------+
+        |Paternal ID |``sa.fam.patID``   |String  |``0``       |
+        |            |                   |        |            |
+        |            |                   |        |            |
+        +------------+-------------------+--------+------------+
+        |Maternal ID |``sa.fam.matID``   |String  |``0``       |
+        |            |                   |        |            |
+        |            |                   |        |            |
+        +------------+-------------------+--------+------------+
+        |Sex         |``sa.fam.isFemale``|Boolean |``N/A``,    |
+        |            |                   |        |``-9``,     |
+        |            |                   |        |or ``0``    |
+        |            |                   |        |            |
+        +------------+-------------------+--------+------------+
+        |Case-control|``sa.fam.isCase``  |Boolean |``0``,      |
+        |phenotype   |                   |        |``-9``,     |
+        |            |                   |        |non-numeric,|
+        |            |                   |        |or the      |
+        |            |                   |        |``missing`` |
+        |            |                   |        |argument, if|
+        |            |                   |        |given       |
+        +------------+-------------------+--------+------------+
+        |Quantitive  |``sa.fam.qPheno``  |Double  |``NA`` or   |
+        |phenotype   |                   |        |the         |
+        |            |                   |        |``missing`` |
```

Comment:
@cseed Are you okay with this table or should it be a bulleted list like in split_multi